### PR TITLE
fix:Add build-tags: [functional] to .golangci.yaml

### PR DESCRIPTION
**Pull request by jingchu000**
Add build-tags: [functional] to .golangci.yaml  And fix the issues caused by the introduction

## Description

Add build-tags: [functional] to .golangci.yaml

## References

#777 

## Review Checklist
- [x] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [x] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [x] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: cmd/openfga/main_test.go [hunk 1]
```diff
@@ -588,6 +589,7 @@ func GRPCCheckTest(t *testing.T, tester OpenFGATester) {
 		t.Run(test.name, func(t *testing.T) {
 
 			storeID := test.input.StoreId
+			//nolint:ineffassign,staticcheck
```
[the remark this answers is not in the document]
Comment (jingchu000) [reply cmd/openfga/main_test.go:592]: image
Deleting will result in an error

[the remark this answers is not in the document]
Comment (jingchu000) [reply cmd/openfga/main_test.go:592]: //Package main contains the root of all commands. If you delete this together, there will be no error message. Do you need to delete it together? I haven't used this syntax before.

[the remark this answers is not in the document]
Comment (miparnisari) [reply cmd/openfga/main_test.go:592]: You can assign the variable in line 602

[the remark this answers is not in the document]
Comment (jingchu000) [reply cmd/openfga/main_test.go]: Thank you. The method you mentioned is feasible,I have already updated.

**Comment by miparnisari** [comment]
Thanks @jingchu000. I see that the CLA step is failing. Can you please make sure that the email that appears in your commit (ingchu@lccomputing.com) matches the email that you use in Github?

**Comment by jingchu000** [comment]
我github的邮箱是904647411@qq.com,我要如何更改呢。

**Comment by jingchu000** [comment]
I executed the CLA signing, but it doesn't seem to work

**Comment by jingchu000** [comment]
image

**Comment by miparnisari** [comment]
@jingchu000 looking at https://github.com/openfga/openfga/commit/d9132818c5dd00fdd8a588f7034a9a58632997c9 

it says `Signed-off-by: jingchu <904647411@qq.com>`

I think you need to do something like `git commit --amend --signoff` to change the sign-off email :)

**Comment by jingchu000** [comment]
@miparnisari ,Thank you very much. It has been approved

**Comment by miparnisari** [comment]
Perfect! Can you merge `main` into your branch?

**Comment by jingchu000** [comment]
@miparnisari  ，Do I merge from the main branch?

**Comment by jingchu000** [comment]
I joined my main branch

**Review by miparnisari — APPROVED** [review]

**Comment by miparnisari** [comment]
@jingchu000 I clicked this button for you to merge `main` into your branch:

image

**Pull request merged**